lib/faucet_pipeline_rails/railtie.rb

```diff
@@ -8,5 +8,13 @@ class Railtie < Rails::Railtie
         Manifest.instance.fetch(source)
       end
     end
+
+    rake_tasks do
+      namespace :assets do
+        task :precompile do
```

Comment:
Please add description to this task, otherwise it won't show up when running `rails -T`